tests/fallbacks/coroutine/test_circuit_breaker.py

```diff
@@ -1,8 +1,10 @@
+from asyncio_toolkit.fallbacks.coroutine.circuit_breaker import (
+    CorotuineCircuitBreaker
```

Comment:
Which? The class name is CorotuineCircuitBreaker
https://github.com/luizalabs/asyncio-toolkit/blob/master/asyncio_toolkit/fallbacks/coroutine/circuit_breaker.py#L6